spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -62,9 +65,6 @@
 import org.springframework.util.Assert;
 import org.springframework.util.StringUtils;
 
-import com.rabbitmq.client.AMQP;
-import com.rabbitmq.client.Envelope;
```

Comment:
We have to come up with the common code style.

SI uses this: https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style#import-statements